src/main/resources/swagger/api-docs

```diff
@@ -0,0 +1,29 @@
+{
+  "basePath": "https://rawls-dev.broadinstitute.org",
```

Comment:
this is going to need to be different for staging and production as well... is there a way to parameterized this?